previewModule.js

```diff
@@ -61,7 +61,17 @@ class PreviewModule{
     }
 
     loginCheck(){
-        let token = localStorage.getItem('token');
+        let token;
+        try {
+            token = localStorage.getItem('token');
+        } catch(e) {
+            document.getElementById('LoginControls').style.display = "none";
```

Comment:
Not a needed change, next time it would be easier putting all these in a single container to hide/show.